INSTALL.rst

```diff
@@ -161,6 +161,22 @@ or other Python packages is not your best option:
   make system-wide Python changes (installing as root or using sudo),
   you can break the operating system.
 
+
+Downloading the entire dataset
```

Comment:
```suggestion
Downloading all demo datasets
```

Reply:
ping @sciunto if you accept this suggestion we can merge the PR :-)

Reply:
I do!